projects/packages/forms/src/dashboard/inbox/export-responses/google-drive.tsx

```diff
@@ -6,19 +6,30 @@ import { isSimpleSite } from '@automattic/jetpack-script-data';
 import { useAnalytics } from '@automattic/jetpack-shared-extension-utils';
 import requestExternalAccess from '@automattic/request-external-access';
 import { Button, Path, Spinner, SVG } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
 import { useCallback, useRef, useState } from '@wordpress/element';
 import { __, _x } from '@wordpress/i18n';
 import clsx from 'clsx';
 /**
  * Internal dependencies
  */
 import { config } from '../..';
-import { useIntegrationStatus } from '../../../blocks/contact-form/components/jetpack-integrations-modal/hooks/use-integration-status';
+import { INTEGRATIONS_STORE } from '../../../store/integrations';
 import { PARTIAL_RESPONSES_PATH } from '../../../util/get-preferred-responses-view';
+/**
+ * Internal dependencies
+ */
+import type { SelectIntegrations, IntegrationsDispatch } from '../../../store/integrations';
+import type { Integration } from '../../../types';
 
 const GoogleDriveExport = ( { onExport, autoConnect = false } ) => {
 	const [ isExporting, setIsExporting ] = useState( false );
-	const { integration, refreshStatus } = useIntegrationStatus( 'google-drive' );
+	const { integration } = useSelect( ( select: SelectIntegrations ) => {
+		const store = select( INTEGRATIONS_STORE );
+		const list = store.getIntegrations() || [];
+		return { integration: list.find( ( i: Integration ) => i.id === 'google-drive' ) };
+	}, [] ) as { integration?: Integration };
+	const { refreshIntegrations } = useDispatch( INTEGRATIONS_STORE ) as IntegrationsDispatch;
```

Comment:
This file had an extra check for if Google was enabled using the useIntegrationStatus hook. Going forward, we'll want to use the new integrations store for single integrations checks like this too. The store allows for immediate checks without an extra endpoint call. You just check if a particular integration slug exists in the list of integrations. I've updated this file to do that. It also now uses the refreshIntegrations() method from the store.